### PR TITLE
chore: Move Storybook to vite build

### PR DESCRIPTION
**Pull request by bdow**

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -8,10 +13,32 @@ module.exports = {
...
+        alias: {
+          src: path.resolve(__dirname, "../src"),
+        },
+      },
+    });
+
+    return {
+      ...mergedConfig,
+      plugins: [
+        // Filter out `vite:react-jsx` per suggestion in `plugin-react`...
```
Comment (stephenh) [inline .storybook/main.js:33]: Is this something we should remove from our main `vite.config.js`?

**Review by stephenh — APPROVED** [review]

File: .storybook/main.js (hunk 1 above)
Comment (bdow) [reply .storybook/main.js:33]: There is no main `vite.config.js` in this repo. Are you talking about `internal-frontend`?

File: .storybook/.babelrc [hunk 2]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (KoltonG) [inline .storybook/.babelrc:1]: Yes! Less Babel!

File: .storybook/main.js [hunk 3]
```diff
@@ -8,10 +13,32 @@ module.exports = {
...
+        logOverride: { "this-is-undefined-in-esm": "silent" },
+      },
+      resolve: {
+        alias: {
+          src: path.resolve(__dirname, "../src"),
+        },
+      },
+    });
+
+    return {
```
Comment (KoltonG) [inline .storybook/main.js:30]: This is great! I think this is what I was missing!

File: package.json [hunk 4]
```diff
@@ -134,6 +133,7 @@
     "ttypescript": "^1.5.12",
     "typescript": "^4.4.2",
     "typescript-transform-paths": "^3.2.1",
+    "vite": "^3.0.9",
```
Comment (KoltonG) [inline package.json:136]: Great! I was wondering if this was Vite 3 compatible! It is!

**Review by KoltonG — COMMENTED** [review]
Love it! I will use this to help setup the Homeowner Checkout repo!

**Pull request merged**